akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -200,3 +200,24 @@ private[akka] final class PropsSink[In](props: Props, val attributes: OperationA
   override protected def newInstance(shape: SinkShape[In]): SinkModule[In, ActorRef] = new PropsSink[In](props, attributes, shape)
   override def withAttributes(attr: OperationAttributes): Module = new PropsSink[In](props, attr, amendShape(attr))
 }
+
+/**
+ * INTERNAL API
+ */
+private[akka] final class TellSink[In](ref: ActorRef, onCompleteMessage: Any,
+                                       val attributes: OperationAttributes,
+                                       shape: SinkShape[In]) extends SinkModule[In, Unit](shape) {
+
+  override def create(materializer: ActorFlowMaterializerImpl, flowName: String) = {
+    val subscriberRef = materializer.actorOf(
+      TellSinkActor.props(ref, materializer.settings.maxInputBufferSize, onCompleteMessage),
```

Comment:
I used an actor because of cancellation when the destination actor terminates, if we don't want that feature we don't need an actor here